.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Feature request
+about: Let us know how we could improve OPA
+title: ''
+labels: feature-request
+assignees: ''
+
+---
+
+<!-- Thanks for opening an issue to request a feature or file a bug!
+If you provide some basic information, it helps us address problems faster. -->
+
+## What functionality would you like to add or improve?
```

Comment:
I think one thing we need to emphasize here is that they should tell _why_ they need something. We don't really need them to weigh in on what specific functionality they think should be added--it's more important tell us _why_ they need something so that we can take that information and come up with the best plan forward (i.e., users don't have as much context as we do.)